src/ai_migrate/utils.py

```diff
@@ -12,6 +13,48 @@ class PRDetails(BaseModel):
     deletions: int
 
 
+@dataclass
```

Comment:
FWIW I am not a fan of having files called utils. It can just mean anything. maybe we can just call this code_handling and move all code handling here? now it just a mixed bag of things max built.

Reply:
point taken.